bitnami/nginx/templates/prometheusrules.yaml

```
@@ -0,0 +1,27 @@
+{{- if and .Values.metrics.enabled .Values.metrics.prometheusRule.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  {{- if .Values.metrics.prometheusRule.namespace }}
+  namespace: {{ .Values.metrics.prometheusRule.namespace }}
+  {{- else }}
+  namespace: {{ .Release.Namespace }}
+  {{- end }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: nginx
+    {{- range $key, $value := .Values.metrics.prometheusRule.selector }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
```

Copy/pasting comments I did at https://github.com/bitnami/charts/pull/8452
Use `app.kubernetes.io/component: metrics`
And deprecate `.Values.metrics.prometheusRule.selector` in favor or `.Values.metrics.prometheusRule.additionalLabels`
```suggestion
    app.kubernetes.io/component: metrics
```